### ISSUE TITLE
Can't import spinalcordtoolbox.reports.qc outside of the scripts directory

### ISSUE THREAD

**Issue by cfhammill** [neuropoly/spinalcordtoolbox#2627]
On current master 4878c3b7 spinalcordtoolbox.reports.qc can't be imported from outside the scripts directory:

```python
import spinalcordtoolbox.reports.qc
```

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/chammill/Documents/spinalcordtoolbox/spinalcordtoolbox/reports/qc.py", line 29, in <module>
    import sct_utils as sct
ModuleNotFoundError: No module named 'sct_utils'
```

This works from in the scripts directory because `sct_utils.py` is in there.

**Comment by jcohenadad** [comment]
good catch! @cfhammill

**Comment by cfhammill** [comment]
I considered moving `sct_utils.py` over to python package directory but it seems there's a substantial amount of purpose built code for finding sct_utils and importing it.

I did a quick dig seeing if there was any easy way to add `sct_utils.py` to `setup.py`, but nothing simple, pretty sure it's doable. As a kludgy solution we could symlink `sct_utils.py` into the `spinalcordtoolbox` directory.

**Comment by jcohenadad** [comment]
> I considered moving sct_utils.py over to python package directory but it seems there's a substantial amount of purpose built code for finding sct_utils and importing it.

I know... this is an epic refactoring that we've been waiting for years. I would suggest to do it step by step, adding unit tests along the way and replacing some `sct_utils` functions by already-existing Python functions (when that applies), and removing those that are not used anymore.

**Comment by cfhammill** [comment]
Sounds like a reasonable approach. Happy to take on the refactor if you think it's the right time.

**Comment by jcohenadad** [comment]
@cfhammill @joshuacwnewton do you know what is the status of this issue?

**Comment by Drulex** [comment]
> @cfhammill @joshuacwnewton do you know what is the status of this issue?

Will be fixed by #2898
